src/runtime_src/hip/api/hipMemory.cpp

```diff
@@ -0,0 +1,221 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Advanced Micro Device, Inc. All rights reserved.
+
+#include <string>
+#include "core/common/error.h"
+#include "hip/config.h"
+#include "hip/core/device.h"
+#include "hip/core/memory.h"
+#include "hip/hip_runtime_api.h"
+
+namespace xrt::core::hip {
+
+// Allocate memory on the device.
+static void
+hipMalloc(void** ptr, size_t size)
+{
+    throw std::runtime_error("Not implemented");
```

Comment:
in XRT , we use 2 spaces . Please update it in all places

Reply:
done.